.github/scripts/check-engines-cli.sh

```diff
@@ -99,7 +101,7 @@ echo "---"
 
 # TODO Add test that makes sure not _wrong_ files are present as well
 # Example: `community-generators (napi, prisma-dbml-generator)` has correct node_modules/prisma/libquery_engine-debian-openssl-1.1.x.so.node, but wrong node_modules/@prisma/engines/query-engine-debian-openssl-1.1.x (also `community-generators (napi, prisma-json-schema-generator)`)
-if [ -f "$qe_location" ]  || [ -f "$qe_location2" ] ; then
+if [ -f "$qe_location" ] || [ -f "$qe_location2" ] || [ $CLI_QUERY_ENGINE_TYPE == "dataproxy" ] ; then
```

Comment:
Should we maybe do a special case here that output "no query engine required as type dataproxy"? I think right now there are actually some files present, that we might not want in the future etc.

Reply:
Will do.